### DISCUSSION TITLE
Is Agalwood Alright

### DISCUSSION THREAD

**Discussion by SonGorLick** [agalwood/Motrix#1300] [General]
Is agalwood still active and or alive, sense i havent seen any updates here in forever

**Comment by ghost** [comment]
已经一年多没有任何一个更新了，这个项目是不是已经被放弃了？

**Reply by phlinsia** [reply]
> Is agalwood still active and or alive, sense i havent seen any updates here in forever

Motrix has no release for a long time since 2021, and I can't find a updated fork on Github. It seems that our maintainer agalwood may have forgotten his password.

**Comment by ImBIOS** [comment]
#1396 He currently focus on his business, but he already have v2 plan for Motrix.

**Comment by shindgew** [comment]
Great News! agalwood just made some commits yesterday!
